src/test/java/de/tum/in/www1/artemis/connector/IrisRequestMockProvider.java

```diff
@@ -78,12 +76,9 @@ public void mockMessageResponse(String responseMessage) throws JsonProcessingExc
             mockServer.expect(ExpectedCount.once(), requestTo(messagesApiURL.toString())).andExpect(method(HttpMethod.POST)).andRespond(withSuccess());
             return;
         }
-        var irisMessage = new IrisMessage();
-        var irisMessageContent = new IrisMessageContent();
-        irisMessageContent.setTextContent(responseMessage);
-        irisMessage.setContent(Collections.singletonList(irisMessageContent));
+        var irisMessage = new IrisMessage(null); // session is not needed for this test
```

Comment:
Then just keep using the empty constructor.
Also it is not specified, because this mocks the response of Pyris, not because it is not needed.